components/GoalsAndMissionCardContent/GoalsAndMissionCardContent.tsx

```diff
@@ -2,196 +2,193 @@ import { Row, Col } from "../bootstrap"
 import Image from "react-bootstrap/Image"
 import styles from "./GoalsAndMissionCardContent.module.css"
 import { SignInWithButton } from "../auth"
+import { useTranslation } from "next-i18next"
 
-const OurGoalsCardContent = () => (
-  <>
-    <h3 className={styles.goalsHeader}>
-      By creating an accessible platform for submitting testimony on legislation
-      in MA via a transparent archive of public testimony, we aim to achieve
-      these goals:
-    </h3>
-    <Row className="mb-4">
-      <Col md={6}>
-        <figure className="text-center mb-3">
+const OurGoalsCardContent = () => {
+  const { t } = useTranslation("goalsandmission")
+  return (
+    <>
+      <h3 className={styles.goalsHeader}>{t("goals.overview")}</h3>
+      <Row className="mb-4">
+        <Col md={6}>
+          <figure className="text-center mb-3">
+            <Image
+              className={styles.imgsize}
+              fluid
+              src="/gov_key.png"
+              alt="government building with key"
```

Comment:
Do we need to translate alt attributes?

Reply:
great point, I'm sure we should! I will capture this in the epic for now